torchvision/prototype/transforms/_type_conversion.py

```diff
@@ -13,7 +13,7 @@ class DecodeImage(Transform):
     _transformed_types = (features.EncodedImage,)
 
     def _transform(self, inpt: torch.Tensor, params: Dict[str, Any]) -> features.Image:
-        return cast(features.Image, F.decode_image_with_pil(inpt))
+        return F.decode_image_with_pil(inpt)  # type: ignore[no-any-return]
```

Comment:
This has to be here, because it seems 

https://github.com/pytorch/vision/blob/c84dbfad97251271a789b252a2a1a52c73f623ff/torchvision/prototype/transforms/functional/_type_conversion.py#L13-L14

doesn't "forward" the type annotations :roll_eyes:

Reply:
It's fine, in all other places we took the decision to silence with ignore rather than cast.

Reply:
Nono, I was just explaining why we need the ignore for future me that is looking confused at the blame why we introduced it in the first place.